docs/migration.md

```diff
@@ -26,6 +26,18 @@ To this:
   })
 ```
 
+And inside of your `configureStore.js` [file](https://github.com/ScriptedAlchemy/redux-first-router-demo/blob/6c8238eee713ce0079aeae1ce328d305bddd0ee3/server/configureStore.js) if you are server side rendering, change this:
+
+ ```js
+  const history = createHistory({ initialEntries: [req.path] })
+  const { store, thunk } = configureStore(history, preLoadedState)
+```
+
+ To this:
+ 
+ ```js
+  const { store, thunk } = configureStore(preLoadedState, [req.path])
```

Comment:
I added the link to `connectRoute` doc. I'm not sure if you mean that an example of `configureStore` should be provided in migration or connectRoute doc. I also added links to commits on `redux-first-router-demo` but they come from the repository owned by ScriptedAlchemy. I don't know if it's a problem or not.

Finally, I gave you the edit right on the fork repo used for this PR, as it will be easier for you to answer those questions and improve this document with your ideas in proper english, which is not my native language.

Reply:
ideally i try to point to the faceyspacey repo for most things, but in the case of pull requests, there was a time where we did operate off of a fork while i got write access to the main repo